tests/SpecTests/ResultExpectation.php

```diff
@@ -126,7 +126,7 @@ public static function fromRetryableReads(stdClass $operation, $defaultAssertion
 
     public static function fromRetryableWrites(stdClass $outcome, $defaultAssertionType)
     {
-        if (property_exists($outcome, 'result')) {
+        if (property_exists($outcome, 'result') && ! self::isErrorResult($outcome->result)) {
```

Comment:
Was this necessary because the retryable writes tests synced in this PR were the first to include error fields under an `outcome.result` object? For example:

```
 "error": true,
"result": {
  "errorLabelsContain": [
    "RetryableWriteError"
  ]
},
```

Reply:
Yes, other tests didn’t include error results.